book/forms.rst

```diff
@@ -1568,6 +1568,15 @@ are 4 possible *parts* of a form that can be rendered:
 
 .. note::
 
+    The portion corresponding to ``collection`` type is ``entry``.
+    In the case of embedded collections, for example,
+    in a form which you have a year which contains months, and months contains days.
+    You want to personalize each collection render, where the form prefix is ``_my_form_year``,
+    you need to write two blocks, ``_my_form_year_months_entry_row`` to customize month render, and
+    ``_my_form_year_months_entry_days_entry_row`` for days inputs.
```

Comment:
Its an example on multiple embedded collections, a form which contains a collectiontype, which contains himself an other collectiontype. If you have an idea for a better explanation, it would help.